x-pack/plugins/monitoring/server/alerts/cpu_usage_alert.ts

```diff
@@ -106,18 +106,17 @@ export class CpuUsageAlert extends BaseAlert {
       this.config.ui.max_bucket_size
     );
     return stats.map((stat) => {
-      let cpuUsage = 0;
       if (this.config.ui.container.elasticsearch.enabled) {
-        cpuUsage =
+        stat.cpuUsage =
           (stat.containerUsage / (stat.containerPeriods * stat.containerQuota * 1000)) * 100;
       } else {
-        cpuUsage = stat.cpuUsage;
+        stat.cpuUsage = stat.cpuUsage;
       }
```

Comment:
I don't think you need this else clause here

Reply:
Updated